docs/api/api_blueprint.apib

```diff
@@ -20834,6 +20834,88 @@ Get the task status and the final result of the task as response
+| `Screening`  | No. of events of type - Screening | Integer |
+| `Game or Competition`  | No. of events of type - Game or Competition | Integer |
+| `Party or Social Gathering`  | No. of events of type - Party or Social Gathering | Integer |
+| `Race or Endurance Event`  | No. of events of type - Race or Endurance Event | Integer |
+| `Meeting or Networking Event`  | No. of events of type - Meeting or Networking Event | Integer |
+| `Attraction`  | No. of events of type - Attraction | Integer |
+| `Class, Training, or Workshop`  | No. of events of type - Class, Training, or Workshop | Integer |
+| `Appearance or Signing`  | No. of events of type - Appearance or Signing | Integer |
+| `Tournament`  | No. of events of type - Tournament | Integer |
+| `Rally`  | No. of events of type - Rally | Integer |
+
+### Show Event Types Statistics [GET]
+
++ Request
+
+    + Headers
+
+            Accept: application/vnd.api+json
+
+            Authorization: JWT <Auth Key>
+
++ Response 200 (application/json)
+
+        {}
+
+
+## Event Topics Statistics Details [/v1/admin/statistics/event-topics]
+
+**Event Topics:**
+
+| Parameter | Description | Type |
+|:----------|-------------|------|
+| `Health & Wellness`  | No. of events of topic - Health & Wellness | Integer |
+| `Home & Lifestyle`  | No. of events of topic - Home & Lifestyle | Integer |
+| `Charity & Causes`  | No. of events of topic - Charity & Causes | Integer |
+| `Other`  | No. of events of topic - Other | Integer |
+| `Religion & Spirituality`  | No. of events of topic - Religion & Spirituality | Integer |
+| `Community & Culture`  | No. of events of topic - Community & Culture | Integer |
+| `Government & Politics`  | No. of events of topic - Government & Politics | Integer |
+| `Auto, Boat & Air`  | No. of events of topic - Auto, Boat & Air | Integer |
+| `Travel & Outdoor`  | No. of events of topic - Travel & Outdoor | Integer |
+| `Hobbies & Special Interest`  | No. of events of topic - Hobbies & Special Interest | Integer |
+| `Sports & Fitness`  | No. of events of topic - Sports & Fitness | Integer |
+| `Business & Professional`  | No. of events of topic - Business & Professional | Integer |
+| `Music`  | No. of events of topic - Music | Integer |
+| `Seasonal & Holiday`  | No. of events of topic - Seasonal & Holiday | Integer |
+| `Film, Media & Entertainment`  | No. of events of topic - Film, Media & Entertainment | Integer |
+| `Family & Education`  | No. of events of topic - Family & Education | Integer |
+| `Science & Technology`  | No. of events of topic - Science & Technology | Integer |
+| `Performing & Visual Arts`  | No. of events of topic - Performing & Visual Arts | Integer |
+| `Food & Drink`  | No. of events of topic - Food & Drink | Integer |
+
+### Show Event Topics Statistics [GET]
+
++ Request
+
+    + Headers
+
+            Accept: application/vnd.api+json
+
+            Authorization: JWT <Auth Key>
+
++ Response 200 (application/json)
+
+        {}
```

Comment:
How about adding a Factory for testing and then adding the expected response here ? Empty response doesn't server the purpose IMO.

Reply:
Since response is dynamic. we cannot add factory here. As which event type/topic and how many of them will be returned.

> def events_past_count(self, obj):
        return get_count(Event.query.filter(Event.ends_at < datetime.now(pytz.utc)))

> Please add one empty line at the end.

There is empty line. If that wasn't there then it is always shown with a red mark